docs/changelog.rst

```diff
@@ -32,6 +32,8 @@ New features:
 
 Bug fixes:
 
+* Fix Spotify API error when tracks do not have audio-features.
+  :bug:`4385`
```

Comment:
Probably no need for an extra bugfix entry in the changelog, since we already have an entry for the new `spotifysync` command itself.

Reply:
Done